### ISSUE TITLE
Improve endless mode heal after win

### ISSUE THREAD

**Issue by stonejess** [thebclickteam/bocracy#220]
Every time you kill an opponent, your health go up so much, you practically can’t die. In addition, it makes the healing tap useless since you heal from your opponents. If we change this, it makes the game harder and you have to strategize.

**Comment by VishwamR** [comment]
Good idea

>

**Comment by ghost** [comment]
That is nice. Possibly, instead of removing this feature, maybe making the auto heal be a lot less (to where it heals very little).

This will also help with users getting so much rewards from endless mode.

**Comment by stonejess** [comment]
I agree, because I sometimes get over 13,000 red backs. Also I’m addicted
